projects/bepro/index.js

```diff
@@ -4,7 +4,7 @@ const { sumTokensExport } = require('../helper/unwrapLPs')
 const config = {
     "ethereum": {
         token: "0xcf3c8be2e2c42331da80ef210e9b1b307c03d36a",
-        bountyNetworks: []
+        bountyNetworks: ["0x695b30d636e4f232d443af6a93df95afd2ff485c"]
```

Comment:
I don't know why this is still showing up as my change. I deleted the old repository and restarted the pull-request on the BEPRO platform. But it still shows the (already merged) changes of my old PR.

Maybe you can tell me what I'm doing wrong. As far as I can see I also cannot close the old PR manually, as it's managed by the bepro-bot.

Reply:
when this happens to me I usually just make a new clone from the repo and continue from the fresh clone; Either way, you can always delete this change locally and commit it and then it'll _not_ show as a change because there's no difference between source and pr